src/main/java/uk/gov/justice/digital/job/DataHubJob.java

```diff
@@ -153,7 +153,7 @@ public void run() {
     private List<Row> getTablesInBatch(Dataset<Row> dataFrame) {
         return dataFrame
                 .select(TABLE, SOURCE, OPERATION)
-                .distinct()
+                .dropDuplicates(TABLE, SOURCE)
```

Comment:
Because the operation was factored into the uniqueness this was returning a list with duplicate tables one for CDC and another for Load.

Reply:
I wonder if we should add some kind of test to prevent regressions

Reply:
Good Idea. Will do so.